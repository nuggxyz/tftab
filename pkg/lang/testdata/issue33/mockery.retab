
GO_PACKAGE = "github.com/walteh/xyz"

pkgs = {
	"pkg/core/v1" = ["CoreInter"]
	"pkg/schema"  = ["SchemaInter"]
}

gen mockery {
	schema = ""
	path   = "../.mockery.yaml"
	data = {
		packages = {
			for pkg, namz in pkgs : "${GO_PACKAGE}/${pkg}" => {
				interfaces = {
					for interfac in namz : interfac => {}
				}
			}
		}
	}
}

