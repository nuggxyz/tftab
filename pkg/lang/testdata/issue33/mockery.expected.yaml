# code generated by retab. DO NOT EDIT.
# join the fight against yaml @ github.com/walteh/retab

# source: "mockery.retab"

packages:
    github.com/walteh/xyz/pkg/core/v1:
        interfaces:
            CoreInter: {}
    github.com/walteh/xyz/pkg/schema:
        interfaces:
            SchemaInter: {}
